inttest/common/launchdelegate.go

```diff
@@ -55,29 +58,37 @@ var _ launchDelegate = (*standaloneLaunchDelegate)(nil)
 // InitController initializes a controller in "standalone" mode, meaning that
 // the k0s executable is launched directly (vs. started via a service manager).
 func (s *standaloneLaunchDelegate) InitController(ctx context.Context, conn *SSHConnection, k0sArgs ...string) error {
```

Comment:
When adding support for stopping and restarting, I asked myself if it's still meaningful to keep the standalone mode around? Is there any situation in which standalone mode would enable something that OpenRC doesn't? Adding the new features somehow felt a bit like reimplementing an init system on the fly.

OTOH, standalone mode could be kept, but not supporting all of the operations and return an error for those that are unsupported.

Reply:
Do you mean for test purposes or in general?


In general I am strictly against dropping standalone mode (if be standalone you mean running binary directly without init system). It would be strange, if we'd claim to be single binary distro and require init system at the same time